owid/walden/index/faostat/2021-08-18/faostat_FS.json

```diff
@@ -5,13 +5,14 @@
   "description": "Suite of Food Security Indicators presents the core set of food security indicators. Following the recommendation of experts gathered in the Committee on World Food Security (CFS) Round Table on hunger measurement, hosted at FAO headquarters in September 2011, an initial set of indicators aiming to capture various aspects of food insecurity is presented here. The choice of the indicators has been informed by expert judgment and the availability of data with sufficient coverage to enable comparisons across regions and over time. Many of these indicators are produced and published elsewhere by FAO and other international organizations. They are reported here in a single database with the aim of building a wide food security information system. More indicators will be added to this set as more data will become available. Indicators are classified along the four dimensions of food security -- availability, access, utilization and stability. For definitions of these indicators, see Definitions and standards below (under Item).",
   "source_name": "Food and Agriculture Organization of the United Nations",
   "url": "http://www.fao.org/faostat/en/#data",
-  "date_accessed": "2021-10-28",
   "file_extension": "zip",
+  "date_accessed": "2022-04-22",
+  "source_data_url": "https://fenixservices.fao.org/faostat/static/bulkdownloads/Food_Security_Data_E_All_Data_(Normalized).zip",
   "license_url": "http://www.fao.org/contact-us/terms/db-terms-of-use/en",
-  "source_data_url": "http://fenixservices.fao.org/faostat/static/bulkdownloads/Food_Security_Data_E_All_Data_(Normalized).zip",
-  "md5": "d2dfbcb4d12706c69b07ca3d5e7e9cc2",
+  "license_name": "CC BY-NC-SA 3.0 IGO",
+  "is_public": true,
   "publication_year": 2021,
   "publication_date": "2021-08-18",
   "owid_data_url": "https://walden.nyc3.digitaloceanspaces.com/faostat/2021-08-18/faostat_FS.zip",
-  "license_name": "CC BY-NC-SA 3.0 IGO"
+  "md5": "93611821b002603fcc29a0b779ad1a38"
```

Comment:
So, in this case their metadata is misleading -- they have in fact made some kind of update since the listed publication date. Arguably we should be publishing this against date accessed instead.

Reply:
I still have a copy of the old file on my machine. I leave it to you whether you want to let it slide this time, or whether you want to make a new index entry for the newly downloaded file.

Reply:
```
lars@juniper ~ » curl -I https://fenixservices.fao.org/faostat/static/bulkdownloads/Food_Security_Data_E_All_Data_\(Normalized\).zip | grep last-modified
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0 1225k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
last-modified: Wed, 06 Apr 2022 09:24:40 GMT
```

Another fairly reliable alternative would be to use the `last-modified` header of the file on their server, rather than the date in their metadata. For the other files, it seems to agree with the publication date.

Reply:
Good idea. I have fetched the modification date as you suggested. For the moment it doesn't make any difference, but I'll change walden so that it looks for a "modification_date" field in the metadata, and if so, then use that [instead of "publication_date" or "publication_year"](https://github.com/owid/walden/blob/master/owid/walden/catalog.py#L213). I'll do this in a different PR.

Reply:
I also downloaded the entire folder before running the ingest script, so I also had a copy of the file. I have put back the old file for 2021-08-18 and reverted the changes on the index file in [my latest commit](https://github.com/owid/walden/pull/27/commits/9a7347829f5ae7fd2e954620ca7e182188103491).